fluentd/configs.d/openshift/output-operations.conf

```diff
@@ -1,5 +1,5 @@
 <match system.var.log** **_default_** **_openshift_** **_openshift-infra_**>
   @type copy
-  @include fluentd_es_ops_config.conf
-  @include fluentd_es_ops_copy_config.conf
+  @include output-es-ops-config.conf
+  @include ../dynamic/es-ops-copy-config.conf
```

Comment:
I suggest to have an `@include output-extra-ops-*.conf` here - then the user could add a config map for `output-extra-ops-splunk.conf` or `output-extra-ops-fluentd-secure-forward.conf` or whatever